cameras/paths.csv

```diff
@@ -13,6 +13,7 @@ Axis,camera,207W,(00:40:8C),rtsp://{{username}}:{{password}}@{{ip_address}}/mpeg
 Axis,camera,P1214-E,(00:40:8C),rtsp://{{username}}:{{password}}@{{ip_address}}/axis-media/media.amp,H.264,554,,,,,,,root,,,http://www.axis.com/files/manuals/um_axis_P1214e_61497r1_en_1412.pdf
 Hikvision,camera,DS-2CD2032-I,(C4:2F:90|C0:56:E3|BC:AD:28|44:19:B6|28:57:BE),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/PSIA/streaming/channels/10{{stream}},H.264,554,1;2,,,,2,1,admin,,,http://www.hikvision.com/UploadFile/image/2013061411054615328.pdf
 Hikvision,nvr,DS-7600 series,(8C:E7:48),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/Streaming/Channels/{{channel}}{{stream}},H.264,554,01;02,1;2;3;4;5;6;7;8,hi-res;lo-res,Channel {{channel}},02,01,admin,12345,,http://www.hikvision.com//uploadfile/image/20150831025213965.PDF
+Hikvision,camera,DS-2CD2142FWD-IW,(00:95:69),rtsp://{{username}}:{{password}}@{{ip_address}}:{{port}}/PSIA/streaming/channels/101,H.264,554,,,,,,,admin,12345,,http://www.hikvision.com/en/Products_accessries_158_i5605.html
```

Comment:
@mdipierro it looks like this matches the rtsp info of row 14, so can you add this to the model as `DS-2CD2032-I;DS-2CD2142FWD-IW` and extend the `oui_regex` of that row instead? (since it also enumerates the streams and uses the `/10{{stream}}` mustache template.